packages/rs-dpp/src/data_contract/document_type/v0/random_document.rs

```diff
@@ -21,11 +30,127 @@ use crate::identity::Identity;
 use crate::prelude::{BlockHeight, CoreBlockHeight, TimestampMillis};
 use crate::version::PlatformVersion;
 use crate::ProtocolError;
-use platform_value::{Bytes32, Identifier};
-use rand::rngs::StdRng;
-use rand::{Rng, SeedableRng};
 
 impl CreateRandomDocument for DocumentTypeV0 {
+    /// Create random documents using json-schema-faker-rs
+    #[cfg(feature = "documents-faker")]
+    fn random_documents_faker(
+        &self,
+        owner_id: Identifier,
+        entropy: &Bytes32,
+        count: u32,
+        platform_version: &PlatformVersion,
+        substitutions: &BTreeMap<&str, Value>,
+    ) -> Result<Vec<Document>, ProtocolError> {
+        use anyhow::Context;
+
+        use crate::document::{
+            extended_document_property_names::FEATURE_VERSION,
+            property_names::{ID, OWNER_ID, REVISION},
+            serialization_traits::DocumentPlatformValueMethodsV0,
+        };
+
+        let json_schema = &self
+            .schema
+            .clone()
+            .try_into()
+            .expect("jsonschema must be a valid JSON");
```

Comment:
Return an error. Schema might not be valid if you create a contract without `full_validation` flag